src/apple/config/raw.rs

```diff
@@ -51,6 +51,8 @@ pub struct Raw {
     pub ios_features: Option<Vec<String>>,
     pub macos_no_default_features: Option<bool>,
     pub macos_features: Option<Vec<String>>,
+    pub ios_version_double: Option<VersionDouble>,
```

Comment:
The field names don't need to be suffixed with `double`; the doubleness is already expressed by the type and doesn't make the name more meaningful.